servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ProtocolConfig.java

```diff
@@ -60,6 +61,35 @@ default String alpnId() {
     @Nullable
     KeepAlivePolicy keepAlivePolicy();
 
+    /**
+     * Get the {@link Http2Settings} that provides a hint for the initial settings. Note that some settings may be
+     * ignored if not supported (e.g. push promise).
+     * @return the {@link Http2Settings} that provides a hint for the initial settings. Note that some settings may be
+     * ignored if not supported (e.g. push promise).
+     */
+    Http2Settings initialSettings();
+
+    /**
+     * Provide a hint on the number of bytes that the flow controller will attempt to give to a stream for each
+     * allocation (assuming the stream has this much eligible data).
+     * <p>
+     * This maybe useful because the amount of bytes the local peer is permitted to write is limited based upon the
+     * remote peer's flow control window for each stream. Some flow controllers iterate over all streams (and may
+     * consider <a href="https://datatracker.ietf.org/doc/html/rfc7540#section-5.3">stream priority</a>) that have data
+     * pending to write and allow them to write a subset of the available flow control window (aka a "quantum"). The
+     * larger this value may result in increased goodput/throughput on the connection, but increase latency on some
+     * streams (if flow control windows become constrained).
+     * @return number of bytes.
+     */
+    int flowControlQuantum();
+
+    /**
+     * Increment to apply to {@link Http2Settings#initialWindowSize()} for the connection stream. This expands the
```

Comment:
From the spec:
> The WINDOW_UPDATE frame can be specific to a stream or to the entire connection.

"for the connection stream" is a bit confusing here. It sounds like the setting increments window for frames with streamId=0. Consider saying that it applies to the entire connection, not a "connection stream".

Also, consider linking to [WINDOW_UPDATE](https://datatracker.ietf.org/doc/html/rfc7540#section-6.9) instead of `Http2Settings#initialWindowSize()`.

Last, please clarify in the javadoc that this is an increment in bytes.